core/primitives/src/runtime/config_store.rs

```diff
@@ -15,8 +15,11 @@ macro_rules! include_config {
 /// the new runtime config in bytes.
 /// Protocol versions are given in increasing order. First one is always 0, so that each version is
 /// mapped to some config.
-static CONFIGS: [(ProtocolVersion, &[u8]); 2] =
-    [(0, include_config!("29.json")), (42, include_config!("42.json"))];
+static CONFIGS: [(ProtocolVersion, &[u8]); 3] = [
+    (0, include_config!("29.json")),
+    (42, include_config!("42.json")),
+    (115, include_config!("115.json")),
```

Comment:
I think we should consider guarding this with nightly protocol feature as well so that we don't forget to change it when it is ready for release.